llvm/test/CodeGen/NVPTX/load-store.ll

```diff
@@ -1,5 +1,7 @@
-; RUN: llc < %s -march=nvptx64 -mcpu=sm_20 | FileCheck %s
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_20 | FileCheck -check-prefixes=CHECK,SM60 %s
```

Comment:
There's a bit of a contradiction between `-mcpu=sm_20` and `-check-prefixes=SM60`.
Perhaps rename the check `NOATOMIC` ?